packages/unchained-client/openapitools.json

```diff
@@ -96,6 +96,19 @@
           "useSingleRequestParameter": true
         }
       },
+      "optimism": {
+        "inputSpec": "https://raw.githubusercontent.com/shapeshift/unchained/af7069aa4ef7a338108bec5ca4faae296d696943/node/coinstacks/optimism/api/src/swagger.json",
```

Comment:
update after unchained pr is merged

Reply:
https://github.com/shapeshift/lib/pull/1157/commits/67f7f8b1d3088b629f1684b5154bf917fc33f495